Rework and consolidate OTLP configuration related pages (#17169)
newrelic/docs-website: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-otlp-troubleshooting.mdx

---
title: New Relic OTLP Troubleshooting
tags:
  - Integrations
  - Open source telemetry integrations
  - OpenTelemetry
  - OTLP
  - Troubleshoot
metaDescription: Troubleshoot common OTLP ingest errors

> freshnessValidatedDate: never

Reviewer: ```suggestion
freshnessValidatedDate: 2024-05-06
```

---

> New Relic has supported [native OTLP ingest](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-otlp/) for several years. In the process of working through support cases that come up from time to time, we've learned about common issues users face. For some, the problem is easy to identify and fix. Others are deviously tricky, given that the internet is unreliable and there are many components (software, networking, hardware, etc) involved under the control of various parties (customers, New Relic, and public networking infrastructure outside the control of either). With so much complexity, configuration, and failure points, it can be difficult to determine which is at fault an how to best address.

Reviewer: ```suggestion
New Relic has supported [native OTLP ingest](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-otlp/) for several years. In the process of working through support cases that come up from time to time, we've learned about common issues users face. For some, the problem is easy to identify and fix. Others are deviously tricky, given that the internet is unreliable and there are many components (software, networking, hardware, etc.) involved under the control of various parties, such as customers, New Relic, and public networking infrastructure outside the control of either. With so much complexity, configuration, and failure points, it can be difficult to determine which is at fault and how to best address.
```

> Filing and working through a support case can be time consuming and at times frustrating for customers (and for New Relic!). Therefore, we've put together this troubleshooting guide to help establish a shared understanding, and provide tools to self-diagnose and fix issues when possible.

Reviewer: ```suggestion
Working through a support case can be time consuming and at times frustrating for customers (and for New Relic!). Therefore, we've put together this troubleshooting guide to help establish a shared understanding, and provide tools to self-diagnose and fix issues when possible.
```

> First, please review the New Relic [OTLP configuration requirements / recommendations](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-otlp/#configuration). It contains essential advice and context that anyone looking to use OTLP with New Relic should be aware of.

Reviewer: @jack-berg This anchor name hasn't been added to the main otlp page. Which section did you intent to link to?

Author: Just the top level. Fixing.
